packages/mdc-typography/README.md

```diff
@@ -65,7 +66,7 @@ We recommend you load Roboto from Google Fonts
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
 </head>
 <body class="mdc-typography">
-  <h1 class="mdc-typography--display4">Big header</h1>
+  <h1 class="mdc-typography--headline1">Big header</h1>
```

Comment:
Suggestion: Remove the "Load Roboto" subheading, since this otherwise works as an overall usage example